libs/MotionKit/tests/MotionKit_test.cpp

```diff
@@ -36,12 +37,14 @@ class MotionKitTest : public ::testing::Test
 	mock::Accelerometer accel {};
 	mock::Gyroscope gyro {};
 
+	mock::Timeout mock_timeout {};
```

Comment:
Add a test using the timeout callback to ensure you stop the motion at end of the timeout